src/main/java/Legin.java

```diff
@@ -0,0 +1,125 @@
+import java.util.Scanner;
+
+public class Legin {
+    private static Task[] tasks = new Task[100];
+    private static int currentTaskCount = 0;
+
+    public static void horizontalLine() {
+        System.out.println("______________________" +
+                "______________________________________");
+    }
+    public static void greet() {
+        horizontalLine();
+        System.out.println("Hello! I'm Legin, your best online companion!");
+        System.out.println("What can I do for you today my friend :D");
+        horizontalLine();
+    }
+    public static void bye() {
+        horizontalLine();
+        System.out.println("Bye " +
+            Character.toString(0x1F44B) +
+            ". Hope to see you again really soon! " +
+            Character.toString(0x1F608));
```

Comment:
Should this indentation for wrapped lines be 8 spaces instead of 4 spaces?

```suggestion
        System.out.println("Bye " +
                Character.toString(0x1F44B) +
                ". Hope to see you again really soon! " +
                Character.toString(0x1F608));
```